### ISSUE TITLE
Learning to Learn

### ISSUE THREAD

**Issue by MOQN** [ml5js/ml5-website-v02-docsify#92]
@QuinnHe @alanvww I suggest to change **Learning** to **Learn** for consistency.

**Comment by alanvww** [comment]
Do you mean this title in the sidebar?
![CleanShot 2024-06-17 at 20 08 35@2x](https://github.com/ml5js/ml5-website-v02-docsify/assets/10086000/b198dd2e-c08e-40e2-a05c-8025dd6e481b)

**Comment by MOQN** [comment]
Yes, it also requires a folder name change and link updates.

<img width="344" alt="Screenshot 2024-06-18 at 8 33 28 AM" src="https://github.com/ml5js/ml5-website-v02-docsify/assets/2150896/df9f0a9e-f066-49e3-8f61-e9a7b1ddd063">

**Comment by alanvww** [comment]
@MOQN I can include this in my next commit or PR. Should we also make the sidebar titles match the page title? 
![CleanShot 2024-06-17 at 21 00 02@2x](https://github.com/ml5js/ml5-website-v02-docsify/assets/10086000/c305c80f-d5b3-47f1-b838-dfedbbe8a89e) If I remember correctly, the page title is the one we should keep.

**Comment by MOQN** [comment]
Yes, good Catch! `BodyPose`, `HandPose`, `FaceMesh`, `NeuralNetwork` (without whitespace). Thank you @alanvww !

**Comment by MOQN** [comment]
@alanvww Could you please uncomment the `SoundClassifier` as well?

```
- [ImageClassifier](/reference/image-classifier.md)
- [SoundClassifier](/reference/sound-classifier.md)
- [Sentiment](/reference/sentiment.md)
```

**Comment by alanvww** [comment]
> @alanvww Could you please uncomment the `SoundClassifier` as well?
> 
> ```
> - [ImageClassifier](/reference/image-classifier.md)
> - [SoundClassifier](/reference/sound-classifier.md)
> - [Sentiment](/reference/sentiment.md)
> ```



> @alanvww Could you please uncomment the `SoundClassifier` as well?
> 
> ```
> - [ImageClassifier](/reference/image-classifier.md)
> - [SoundClassifier](/reference/sound-classifier.md)
> - [Sentiment](/reference/sentiment.md)
> ```

Surely! Doing it now :D

**Comment by alanvww** [comment]
@MOQN Fixed the naming with #98, please check! I also added the style to animate the transformation of the sidebar toggle movement. Now it will move consistently with the sidebar :)
